app/src/main/java/org/mozilla/fenix/sync/SyncTabItemMenu.kt

```diff
@@ -0,0 +1,42 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.sync
+
+import android.content.Context
+import mozilla.components.browser.menu.BrowserMenuBuilder
+import mozilla.components.browser.menu.item.SimpleBrowserMenuItem
+import org.mozilla.fenix.R
+import org.mozilla.fenix.library.LibraryItemMenu
+
+class SyncTabItemMenu(
+    private val context: Context,
+    private val onItemTapped: (Item) -> Unit = {}
+) : LibraryItemMenu {
+    sealed class Item {
+        object Copy : Item()
+        object Share : Item()
+        object OpenInNewTab : Item()
+        object OpenInPrivateTab : Item()
+    }
+
+    override val menuBuilder by lazy { BrowserMenuBuilder(menuItems) }
+
+    private val menuItems by lazy {
+        listOfNotNull(
+            SimpleBrowserMenuItem(context.getString(R.string.history_menu_copy_button)) {
```

Comment:
We should probably create new string resources here, otherwise our content descriptions will be referencing "history" items